apps/shared/models/dto/user/userPublic.dto.ts

```diff
@@ -0,0 +1,13 @@
+export class UserPublicDTO {
+  readonly userid: number;
+  readonly username: string;
+  readonly role: string;
+  readonly active: boolean;
+
+  constructor(userId: number, userName: string, userRole: string, userIsActive: boolean) {
+    this.userid = userId;
```

Comment:
Case should be consistent :
```suggestion
  readonly userId: number;
  readonly username: string;
  readonly role: string;
  readonly active: boolean;

  constructor(userId: number, userName: string, userRole: string, userIsActive: boolean) {
    this.userId = userId;
```

Reply:
Or should it be just named `id` instead ?